wiki/BanchoBot/en.md

```diff
@@ -1,28 +1,164 @@
 
-
-<!-- BanchoBot (server) commands start with an exclamation mark! `!`-->
 ## BanchoBot Commands
 
-To use BanchoBot commands, you must use the `!` prefix everytime. **Normal users can only use these commands in multiplayer channels and in BanchoBot PM.** If a user send a command in public chat, normal users won't see it and they get it as a private message.
-Users can also use `/bb <command>` client command to automatically open a BanchoBot tab and sending the command immediately.
-
-*For a list of slash (client) commands, visit the [Chat Console](/wiki/Chat_Console#commands-list) article.*
-
-<!-- Commands List -->
-| Command   | Argument(s)            | Description | Example Input | Example Response |
-| --------- | ---------------------- | ----------- | ------------- | ---------------- |
-| `HELP`    | None                   | Shows the list of available commands                                                    | `!help`                               | This list |
-| `ROLL`    | `[number]`             | Randomly gets a value up to the selected number (default is 100).                       | `!roll 1000`                          | "Nathanael rolls 789 point(s)" |
-| `STATS`   | `<user_name>`          | Displays the stats of the user depending on the mode they're playing                    | `!stats peppy`                        | "Stats for [peppy](https://osu.ppy.sh/u/2): <br> Score:    412,018,739 (#94718) <br> Plays:    7073 (lv65) <br> Accuracy: 87.31%" |
-| `WHERE`   | `<user_name>`          | Shows the current location of the user.                                                 | `!where Kyubey`                       | "Kyubey is in Russia" |
-| `FAQ`     | `[lang:]<entry>/list`  | Displays the message of the entry. Use `list` as entry to show all available entries.   | `!faq ping`                           | "Pong!" |
-| `REPORT`  | `<user_name> <reason>` | Sends a report to the chat moderators. Replace username spaces to underscores `_`       | `!report S_o_h spamming in #japanese` | "Chat moderators have been alerted. Thanks for your help." |
-| `REQUEST` | None                   | Receive a random beatmap that a user requests modding assistance.                       | `!request`                            | "[HoneyWorks - Tokyo Summer Session feat. CHiCO](https://osu.ppy.sh/s/426252) by MrSergio" |
-<!-- Note: The !search command is not included because it is no longer working. -->
+*For a list of in-game client commands, see: [Chat Console](/wiki/Chat_Console#commands-list)*
+
+BanchoBot commands start with `!` immediately followed by the command. These commands work in-game and in external IRC clients and can be used in multiplayer chat channels and in BanchoBot's PM (private message). If a normal user sends a BanchoBot command in a public chat channel, other players will not see it, but the response will be displayed in a private message from BanchoBot.
```

Comment:
immediately -> directly?
These commands work in-game and in external IRC clients and can be used in multiplayer chat channels and in BanchoBot's PM (private message). -> These commands work in-game and in external IRC clients. They can be used in multiplayer chat channels and in BanchoBot's PM (private message).
other players -> other users